features/environment.py

```diff
@@ -901,12 +912,13 @@ def clone(self, from_name, cluster_name, to_name):
                 }
             }
         }
-        self.start(to_name, custom_config=custom_config)
+        self.start(to_name, custom_config=custom_config, max_wait_limit=max_wait_limit)
```

Comment:
Let's not make assumptions about default value of long_running = 40:
```suggestion
        kwargs = {'custom_config': custom_config}
        if long_running:
            kwargs['max_wait_limit'] = -1
        self.start(to_name, **kwargs)
```